idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "git"
+version: "5.3.0"
```

Comment:
Version: "git" is intentional.  Please revert this commit.  The "git" version is pulled from the release tag when I publish a release using github actions.

Reply:
My bad, it just resulted in an issue when adding frogfs as a ESP component via submodule and wanting to point to branch head. I remove the commit.